[IMP] l10n_be_codabox: remove limitation for accountant (#11993)
odoo/documentation: content/applications/finance/fiscal_localizations/belgium.rst

CodaBox
-------

> **CodaBox** is a service that allows Belgian companies and accounting firms to access to bank

Author: Hihi! If we remove the 'to' we still don't need to add an s: "Codabox is a service that allows Belgian companies and accounting firms to access bank information and statements."

information and statements. Odoo provides a way to import such statements automatically.

Configuration
~~~~~~~~~~~~~

.. _belgium/codabox-configuration-connection:

> Configure the connection

Reviewer: Connection

************************

> You must first :ref:`install <general/install>` :guilabel:`CodaBox` to start.

Reviewer: I would actually move this under Configuration since it's a prerequisite for everything under the Configuration section. I would also slightly rephrase it to make it clear people must install the *module*:
```suggestion
To configure and use Codabox, first :ref:`install <general/install>` the :guilabel:`CodaBox` module.
```

.. tabs::

>    .. tab:: For Companies

Reviewer: ```suggestion
   .. tab:: For companies
```

      .. important::

>          Make sure the company settings are correctly configured, i.e., the country is set to
>          :guilabel:`Belgium`, the :guilabel:`Tax ID` or :guilabel:`Company ID` field is filled.

Reviewer: ```suggestion
         Make sure the :doc:`company settings </applications/general/companies>` are correctly configured, i.e., the country is set to :guilabel:`Belgium` and the :guilabel:`Tax ID` or :guilabel:`Company ID` field is filled.
```

      #. Go to :menuselection:`Accounting --> Configuration --> Settings`, then go to the

>          :guilabel:`CodaBox` section.

Reviewer: ```suggestion
         :guilabel:`CodaBox & SODA` section.
```

      #. Click on :guilabel:`Manage Connection` to open the connection wizard, which shows the
         :guilabel:`Company VAT/ID` number that will be used for the connection.
      #. If this is your **first connection**, click on :guilabel:`Create connection`.
         The wizard confirms that the connection has been created on **Odoo's side**. Follow the
         steps to validate the connection on **CodaBox's side** too.

         If this is **not your first connection**, the :guilabel:`Password` provided by
         Odoo during the first connection will be requested to create a new connection.

            .. note::
               This :guilabel:`Password` is unique to Odoo and must be stored securely
               on your side.

>    .. tab:: For Accounting Firms

Reviewer: ```suggestion
   .. tab:: For accounting firms
```

      .. note::

>          The accounting firm must manage their clients on separate databases and configure them

Reviewer: ```suggestion
         Accounting firms must manage their clients on separate databases and configure them
```

         individually to avoid mixing up their data. The connection must be made by an accounting
         firm with valid CodaBox Connect credentials.
         In the following instructions, we will refer to your client's company as *Company* and to
         your accounting firm as *Accounting Firm*.

      .. important::

>          Make sure the company settings are correctly configured, i.e., the country is set to
>          :guilabel:`Belgium`, the :guilabel:`Tax ID` or :guilabel:`Company ID` and
>          :guilabel:`Accounting Firm` fields are filled, as well as the :guilabel:`Tax ID` of the
>          :guilabel:`Accounting Firm`.

Reviewer: ```suggestion
         Make sure the :doc:`company settings </applications/general/companies>` are correctly configured, i.e.,      the country is set to :guilabel:`Belgium`, the :guilabel:`Tax ID` or :guilabel:`Company ID` and
         :guilabel:`Accounting Firm` fields are filled, as well as the :guilabel:`Tax ID` of the
         :guilabel:`Accounting Firm`.
```

>       #. Go to :menuselection:`Accounting --> Configuration --> Settings`, then go to the
>          :guilabel:`CodaBox` section.

Reviewer: ```suggestion
      #. Go to :menuselection:`Accounting --> Configuration --> Settings`, then go to the
         :guilabel:`CodaBox & SODA ` section.
```
